pretty.js

```diff
@@ -62,7 +75,31 @@ function filter (value, messageKey, eol, excludeStandardKeys) {
   var result = ''
 
   for (var i = 0; i < keys.length; i++) {
-    if (filteredKeys.indexOf(keys[i]) < 0) {
+    if (errorLikeObjectKeys.indexOf(keys[i]) !== -1) {
+      var arrayOfLines = ('    ' + keys[i] + ': ' + withSpaces(JSON.stringify(value[keys[i]], null, 2), eol) + eol).split('\n')
+
+      for (var j = 0; j < arrayOfLines.length; j++) {
+        if (j !== 0) {
+          result += '\n'
+        }
+
+        var line = arrayOfLines[j]
+
+        if (/^\s*"stack"/.test(line)) {
+          var matches = /^(\s*"stack":)\s*"(.*)",?$/.exec(line)
+
+          if (matches.length === 3) {
+            var indentSize = /^\s*/.exec(line)[0].length + 4
+            var indentation = indent(indentSize)
```

Comment:
can you remove the `indent` function and inline it here?

Reply:
Done